packages/client/hmi-client/src/components/code/tera-code.vue

```diff
@@ -48,20 +48,31 @@
 						/>
 					</template>
 					<template v-else>
+						<Button
+							label="Create model from code"
+							@click="isModelNamingModalVisible = true"
+							:loading="isCodeToModelLoading"
+						/>
 						<Button label="Download Zip" />
 					</template>
 				</section>
 			</section>
 		</template>
-		<v-ace-editor
-			v-model:value="codeText"
-			@init="initialize"
-			:lang="programmingLanguage"
-			theme="chrome"
-			style="height: 100%; width: 100%"
-			class="ace-editor"
-			v-if="programmingLanguage !== ProgrammingLanguage.Zip"
-		/>
+		<div v-if="programmingLanguage !== ProgrammingLanguage.Zip" class="flex h-full">
```

Comment:
My eyes, it burns!!! Those classes needs to go... in hell!!!

Reply:
changed!